src/js/components/Select/__tests__/Select-test.js

```diff
@@ -307,4 +307,23 @@ describe('Select', () => {
     expect(container.firstChild).toMatchSnapshot();
     expect(document.getElementById('test-select__drop')).toBeNull();
   });
+
+  ['small', 'medium', 'large'].forEach(size => {
```

Comment:
Can we call this variable `dropHeight` instead of `size`, to avoid confusion with the `size` property.